extensions/control-plane/api/management-api/contract-negotiation-api/src/main/java/org/eclipse/edc/connector/controlplane/api/management/contractnegotiation/transform/JsonObjectFromContractNegotiationTransformer.java

```diff
@@ -64,8 +66,12 @@ public JsonObjectFromContractNegotiationTransformer(JsonBuilderFactory jsonFacto
                 .add(CONTRACT_NEGOTIATION_COUNTERPARTY_ID, contractNegotiation.getCounterPartyId())
                 .add(CONTRACT_NEGOTIATION_COUNTERPARTY_ADDR, contractNegotiation.getCounterPartyAddress())
                 .add(CONTRACT_NEGOTIATION_CALLBACK_ADDR, callbackAddresses)
+                .add(CONTRACT_NEGOTIATION_CORRELATION_ID, contractNegotiation.getCorrelationId())
                 .add(CONTRACT_NEGOTIATION_CREATED_AT, contractNegotiation.getCreatedAt());
 
+        contractNegotiation.getContractOffers().stream()
+                .findAny()
+                .ifPresent(contractOffer -> builder.add(CONTRACT_NEGOTIATION_ASSET_ID, contractOffer.getAssetId()));
```

Comment:
Maybe better to use the `lastOffer` method. Multiple offers for negotiation is not really supported, but that method is the one actually used.